src/coreclr/nativeaot/Runtime/MiscHelpers.cpp

```diff
@@ -37,6 +37,8 @@
 #include "RhConfig.h"
 #include <minipal/cpuid.h>
 
+#include "../../utilcode/yieldprocessornormalized.cpp"
```

Comment:
Maybe it would make sense to include this into the naot version of yieldprocessornormalized.cpp